x-pack/plugins/apm/server/routes/services/get_service_metadata_details.ts

```diff
@@ -212,6 +213,7 @@ export async function getServiceMetadataDetails({
           faasTriggerTypes: response.aggregations?.faasTriggerTypes.buckets.map(
             (bucket) => bucket.key as string
           ),
+          hostArchitecture: host?.architecture,
```

Comment:
is it possible that a service could have N lambda functions with different architectures? 

in that case, which architecture do we display in the popup?

Reply:
Can you please elaborate why the recent one cause it's a little bit confusing to me 🙈, usually in the popup  we display summary metadata for the service. 

when I see the following:
![image](https://user-images.githubusercontent.com/3369346/198326399-1e6b79a6-d542-4adc-8c10-482daf676dfd.png)

I expect both functions have `arm` architecture. (but it can be different)

I think, for the popup can be solved with simple `terms` agg on the `host.architecture`.

Reply:
I'm afraid if we show both the user might be confused about the estimated cost. As we use the most recent one there. @AlexanderWert WDYT? should we show a list of architecture here or just the most recent?

Reply:
The point about the `recent one`  was in the case that someone **changes** the architecture a specific function.
But if we have an APM service with multiple functions, I think we should show both here.

TBH I haven't thought about the cost estimation for a service that has multiple different functions (with different architectures).

I think, in this case we would need to calculate the cost for each function individually and then sum up the result. Not only because of the architecture but also because they might have different memory sizes, which also influences the costs significantly. Not sure if we are doing this here already or how complicated that would be?

Reply:
Agreed with @AlexanderWert that I'll try to tackle this on another PR.